docs/installation.rst

```diff
@@ -0,0 +1,185 @@
+.. meta::
+   :description: A library that provides GEMM operations with flexible APIs and extends functionalities beyond the traditional BLAS library
+   :keywords: hipBLASLt, ROCm, library, API, tool
+
+.. _installation:
+
+***********************
+Installation
+***********************
+
+This document provides all the information required to build and install hipBLASLt on Linux systems.
+
+Prerequisites
+=============
+
+* A ROCm enabled platform, for more information refer to `ROCm Documentation <https://rocm.docs.amd.com/>`_.
+* A compatible version of hipBLAS
+
+Installing pre-built packages
+=============================
+
+Download pre-built packages either from `ROCm's package servers <https://rocm.github.io/install.html#installing-from-amd-rocm-repositories>`_ or by clicking the GitHub releases tab and manually downloading, which could be newer.  Release notes are available for each release on the releases tab.
```

Comment:
The link <https://rocm.github.io/install.html#installing-from-amd-rocm-repositories>` is not correct. Please refer rocm.docs.amd.com and provide the correct link. Also, please review the entire paragraph on installing prebuilt packages and ensure that it is still valid.

Reply:
@yhuiYH What are we asking them to download from here: https://github.com/ROCm/hipBLAS/releases? This page has release notes.

Reply:
@SwRaw  It should be https://github.com/ROCm/hipBLASLt/releases but not https://github.com/ROCm/hipBLAS/releases

Reply:
@jichangjichang Ok. But it still has release notes only. We are asking users to manually download packages from this page.

Reply:
@jichangjichang Can we address this please: "https://github.com/ROCm/hipBLAS/releases has no packages to be downloaded. Should I remove the line where we ask them to download?" 
Ref:
Installing prebuilt packages
=============================

Download prebuilt packages either from `ROCm's native package manager <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html#native-package-manager>`_ or by going to the `GitHub releases <https://github.com/ROCm/hipBLAS/releases>`_ and manually downloading, which could be newer.  Release notes are available for each release on the releases tab.

Reply:
Done!
@lpaoletti Please review.